server/src/controllers/rewardController.js

```diff
@@ -1,5 +1,25 @@
 const { getRewardModel } = require("../database");
 
+/**
+ * Filter Body
+ * @param req the HTTP Request object
+ * @returns {Object} An object with only the parameters we need from the request body
```

Comment:
I'd be interested in chatting about how this works.

Reply:
@quinnfinite You'd have to ask Zach, I inherited it from him, but I think what's happening here is that it's like a destructuring of the response body so that you can access things easier? I'll. actually ask him and I can tell you what I find out.